apply-templates.sh

```diff
@@ -13,14 +13,6 @@ elif [ "$BASH_SOURCE" -nt "$jqt" ]; then
 	wget -qO "$jqt" 'https://github.com/docker-library/bashbrew/raw/9f6a35772ac863a0241f147c820354e4008edf38/scripts/jq-template.awk'
 fi
 
-jqf='.template-helper-functions.jq'
```

Comment:
https://github.com/docker-library/rabbitmq/blob/d8d2d5317d507cfe67c2b7cba9ffa92bb5526e83/.gitignore#L2 :eyes:

(we should remove it from `.gitignore` too :+1:)